### DISCUSSION TITLE
Robo Vacuum Status On when not running.

### DISCUSSION THREAD

**Discussion by darrenlim86** [iklein99/homebridge-smartthings#109] [General]
I think when robo vacuum is charging the status should be off.

**Comment by iklein99** [comment]
The status shown is what is reported by the device through Smartthings.

**Reply by darrenlim86** [reply]
I am using Samsung Powerboat R7040 and homebridge keeps telling me that the robot vacuum is "On". When I toggle the switch on Home app, the robot vacuum just beeps. Any ideas?

**Comment by iklein99** [comment]
@darrenlim86 , the plugin doesn't support robot vacuums yet.  The Smartthings capabilities for these devices are listed as proposed at this point and non-production.  That said, I can see if we can support them, but i need some help from you.  I need to see the detailed logs when homebridge starts up so I can read the capabilities reported by the device.

Follow these steps:

1. Truncate your logs
2. Go into Homebridge settings and set the debug flag to on.
3. Restart Homebridge
4. Wait about 30 seconds.  Download the logs
5. Go back into homebridge settings and reset the debug flag
6. Restart homebridge
7. Send the log file to me at iklein99@comcast.net

Thanks.
